docs/sources/enterprise/saml.md

```diff
@@ -109,6 +117,55 @@ Grafana provides configuration options that let you modify which keys to look at
 
 An example is `assertion_attribute_name = "givenName"` where Grafana looks within the assertion for an attribute with a friendly name or name of `givenName`. Both, the friendly name (e.g. `givenName`) or the name (e.g. `urn:oid:2.5.4.42`) can be used interchangeably as the value for the configuration option.
 
+### Configure team sync
```

Comment:
```suggestion
### Configure Team Sync
```

Reply:
We don't have **Team Sync** in UI and at the other pages we use **Team sync** in sentence case.